migration/migrating-3-4/migrating-openshift-3-to-4.adoc

```diff
@@ -20,12 +21,12 @@ Before you begin your migration, be sure to review the information on xref:../mi
 
 .Prerequisites
 
-* The source cluster must be {product-title} 3.7 or later.
+* The source cluster must be {product-title} 3.7, 3.9, 3.10, or 3.11.
 * The target cluster must be {product-title} 4.2.
-* You must have xref:../cli_reference/openshift_cli/administrator-cli-commands.html#policy[`cluster-admin` privileges] on all clusters.
+* You must have xref:../../cli_reference/openshift_cli/administrator-cli-commands.html#policy[`cluster-admin` privileges] on all clusters.
 * You must have `podman` installed.
```

Comment:
`podman` was really only required for 3.x clusters because you need to run and extract an `operator.yml` file off the container to install the operator. With 4.x clusters, OLM is installed with our metadata, so podman isn't required.

Reply:
Good catch. That wasn't supposed to be there.